rpm/0006-Remove-type-query-argument-for-dropbox.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/oauth2plugin.cpp
++++ b/src/oauth2plugin.cpp
+@@ -140,7 +140,8 @@ void OAuth2Plugin::sendOAuth2AuthRequest()
+     if (!d->m_oauth2Data.Display().isEmpty()) {
+         url.addQueryItem(DISPLAY, d->m_oauth2Data.Display());
+     }
+-    url.addQueryItem(QString("type"), d->m_mechanism);
++    if (!d->m_oauth2Data.Host().contains(QLatin1String("dropbox.com")))
```

Comment:
According to https://www.dropbox.com/developers/core/docs some of the new API calls require a "dropboxapi.com" host.  So, this check should account for that one also.